{"5abf18a073330975d0499b6d3bc096cd":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"fac7d1f428bf88e7","reportUrl":"https://canonical.github.io/sdcore-tests/2278//#testresult/fac7d1f428bf88e7","status":"broken","statusDetails":"TimeoutError: Timed out waiting for Juju model to be ready!","time":{"start":1739327185183,"stop":1739328492555,"duration":1307372}}]},"3be1fd6b051e2a7f16274330c84676d8":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"4102b4cb01476ca4","reportUrl":"https://canonical.github.io/sdcore-tests/2278//#testresult/4102b4cb01476ca4","status":"broken","statusDetails":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='grafana-0.grafana-endpoints.cos-lite.svc.cluster.local', port=3000): Max retries exceeded with url: /api/search?query=5G%20Network%20Overview (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x74c896eb12d0>: Failed to resolve 'grafana-0.grafana-endpoints.cos-lite.svc.cluster.local' ([Errno -3] Temporary failure in name resolution)\"))","time":{"start":1739328785562,"stop":1739328788569,"duration":3007}}]},"33090515e695e4b8830c4d8ff5c85a7d":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":1},"items":[{"uid":"9bb86c4efd250975","reportUrl":"https://canonical.github.io/sdcore-tests/2278//#testresult/9bb86c4efd250975","status":"passed","time":{"start":1739328492564,"stop":1739328785545,"duration":292981}}]},"4f63e90853beae7da6f5a54ef6170a8c":{"statistic":{"failed":0,"broken":0,"skipped":1,"passed":0,"unknown":0,"total":1},"items":[{"uid":"c926225ae02da86a","reportUrl":"https://canonical.github.io/sdcore-tests/2278//#testresult/c926225ae02da86a","status":"skipped","statusDetails":"Skipped: Traefik issue: https://github.com/canonical/traefik-k8s-operator/issues/361","time":{"start":1739328785549,"stop":1739328785549,"duration":0}}]}}